### PR TITLE
Style tree content to full width

### PR DESCRIPTION
**Pull request by chengcyber**
First of all, thank you for your contribution! 😄

New feature please send pull request to feature branch, and rest to master branch.
Pull request will be merged after one of collaborators approve.
Please makes sure that these form are filled before submitting your pull request, thank you!

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [ ] Other (about what?)

### What's the background?

It's not elegant to control tree content width by calculating. Like the Tree draggable demo https://ant.design/components/tree/#components-tree-demo-draggable

```
/* You can add the following CSS to your project to make draggable area bigger */
#components-tree-demo-draggable .draggable-tree .ant-tree-node-content-wrapper {
  width: calc(100% - 18px);
}
``` 

### Changelog description (Optional if not new feature)

make tree content full width. You can preview the style works on codesandbox

[Edit k29zxwrnxr](https://codesandbox.io/s/k29zxwrnxr)

image

### Self Check before Merge

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

### Additional Plan? (Optional if not new feature)

> If this PR related with other PR or following info. You can type here.

### CODE REVIEW

**Comment by yesmeck** [comment]
It's not horizontal aligned.

**Comment by afc163** [comment]
We can add `<Tree blockNode>` prop for this, cause it could be a breaking change of style.

**Comment by kimochg** [comment]
According to the discussion, it seems to me the todo list is as follow:

- new props `blockNode` in `rc-tree`
- add styles for `.ant-tree.block-node`

I am not pretty sure about it. Could anyone give a bright routine of achieving this full-width tree content proposal?

**Comment by afc163** [comment]
`<Tree blockNode>` renders as `.ant-tree.ant-tree-block-node`.

**Comment by afc163** [comment]
@kimochg Can we make this implementation?

```jsx
<Tree blockNode />
```

**Comment by kimochg** [comment]
@afc163 done

**Comment by zombieJ** [comment]
pls help to update doc.
(Add one column for version info. ref: [here](https://github.com/ant-design/ant-design/pull/15055/files#diff-465bce79248581e6b69eaef8984b9ebf))

**Comment by kimochg** [comment]
@zombieJ docs updated. plz check, thanks.

File: components/tree/index.zh-CN.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ subtitle: 树形控件
 | 参数 | 说明 | 类型 | 默认值 |
 | --- | --- | --- | --- |
 | autoExpandParent | 是否自动展开父节点 | boolean | true |
+| blockNode | 是否树节点填充剩余空间 | boolean | false |
```
Comment (afc163) [inline components/tree/index.zh-CN.md]: 节点占据一行。

**Comment by zombieJ** [comment]
Feature branch updated. Pls merge ~

**Review by afc163 — APPROVED** [review]

**Pull request merged**